src/data/dynamic/form/Judge.js

```diff
@@ -48,6 +48,8 @@ export const FIELDS = {
     maxLength: 50,
     width: 12,
     required: true,
+    regexValidation:
+      /^(\+\d{1,2}\s?)?1?\-?\.?\s?\(?\d{3}\)?[\s.-]?\d{3}[\s.-]?\d{4}$/,
```

Comment:
make a new regex file, since these regex statements are long and not as easy to read cause regex is hard to read, it can be very easy to mess this up across multiple forms